codegen/method.go

```diff
@@ -342,6 +349,8 @@ func (ms *MethodSpec) setExceptions(
 				Message: e.Name,
 			},
 		}
+		ms.Exceptions[i] = exception
```

Comment:
You might need to sort Exceptions at the end of constructing it to ensure that the ordering in the templates is fixed. I'm not sure what ordering resultSpec.Exceptions will be in.